### PR TITLE
Add an initial roadmap for plumbing

### PR DESCRIPTION
**Pull request by afrittoli**
# Changes

Add an initial roadmap for plumbing

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: roadmap.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Tekton Plumbing 2020 Roadmap
+
+This is an incomplete list of work we hope to accomplish in 2020.
+
+Highlights:
+
+- [Release Automation](#release-automation)
+- [Tekton Based CD](#tekton-base-cd)
```
Comment (dibyom) [inline roadmap.md:8]:
```suggestion
- [Tekton Based CD](#tekton-based-cd)
```

File: roadmap.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+- [Tekton Based CI](#tekton-base-ci)
```
Comment (dibyom) [inline roadmap.md:9]:
```suggestion
- [Tekton Based CI](#tekton-based-ci)
```

File: roadmap.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+## Release Automation
+
+We do nightly releases for each Pipeline, Triggers and Dashboard. We deploy them
```
Comment (dibyom) [inline roadmap.md:13]:
```suggestion
We do nightly releases for  Pipeline, Triggers, and Dashboard. We deploy them
```

**Comment by vdemeester** [comment]
ping @afrittoli

**Comment by bobcatfish** [comment]
Looks good to me!

Do we want to include anything around:
* migrating off of prow, or what parts of Prow we'd want to migrate off of?
* migrating from bash scripts to Tekton Tasks (maybe a subset of the previous)

**Comment by vdemeester** [comment]
/lgtm

Let's get this in and do follow-ups :angel: (@bobcatfish @dibyom)

**Pull request merged**